core/src/main/java/org/bitcoinj/crypto/DeterministicHierarchy.java

```diff
@@ -58,7 +58,7 @@ public DeterministicHierarchy(DeterministicKey rootKey) {
     }
 
     /**
-     * Inserts a key into the heirarchy. Used during deserialization: you normally don't need this. Keys must be
+     * Inserts a key into the hierarchy. Used during deserialization: you normally don't need this. Keys must be
```

Comment:
Since this is a common misspellung, and also to not touch this file in this PR, I just fixed this misspelling in the entire project. If you rebase on master, this change will disappear.